app/controllers/hxlstats_controller.rb

```diff
@@ -0,0 +1,69 @@
+class HxlstatsController < ApplicationController
+  # Class to collect and aggregate data to parse to the HXL proxy/endpoint
+  def new
+  end
+
+  def create
+  end
+
+  # rubocop:disable MethodLength
+  def show
+    # Prepare output results array
+    @results = []
+    @results.append(["Emotional State", "Stage of journey", "Country drawn in", "Total children affected",
```

Comment:
When to have spaces in the tags and when not?

Deferring named constants to when they are actually used in multiple places.